src/app/admin/components/change-author/change-author.component.ts

```diff
@@ -32,25 +33,15 @@ export class ChangeAuthorComponent implements OnInit {
   constructor(
     private http: HttpClient,
     private authorshipService: AuthorshipService,
+    private learningObjectServie: LearningObjectService,
```

Comment:
```suggestion
    private learningObjectService: LearningObjectService,
```

If you make this change all references will have to be updated too.